client/src/types/typesGraphs.ts

```diff
@@ -45,32 +45,32 @@ export interface GraphNodeDataInterface {
 }
 
 export interface GraphNodeInterface {
-id: string,
-concept?: string,
-label: string,
-nodeType?: string,
-nodeSubType?: string,
-dataType?:string,
-data?: GraphNodeDataInterface,
-metadata?: any,
+  id: string,
+  concept?: string,
+  label: string,
+  nodeType?: string,
+  nodeSubType?: string,
+  dataType?:string,
+  data?: GraphNodeDataInterface,
+  metadata?: any,
 }
 
 export interface GraphEdgeInterface {
-id?: string,
-source: string,
-target: string,
-edgeType?: string,
-data?: any
+  id?: string,
+  source: string,
+  target: string,
+  edgeType?: string,
+  data?: any
 }
 
 export interface GraphInterface {
-nodes: GraphNodeInterface[],
-edges: GraphEdgeInterface[],
+  nodes: GraphNodeInterface[],
+  edges: GraphEdgeInterface[],
 }
 
 export interface ModelGraphInterface {
-abstract: GraphInterface,
-detailed: GraphInterface
+  abstract: GraphInterface,
```

Comment:
We are not going to need this interface anymore because now we have different modeling frameworks (Petri Nets, GrFNs, etc.)